### DISCUSSION TITLE
Many .gguf models can't be loaded with Llama-cpp

### DISCUSSION THREAD

**Discussion by JasonS09** [oobabooga/text-generation-webui#5473] [Q&A]
I'm trying to load `laser-dolphin-mixtral-2x7b-dpo.Q2_K.gguf` with Llama-cpp using this UI. But I get the following error immediatelly:

`AttributeError: 'LlamaCppModel' object has no attribute 'model'`

This is not the only model I can't work with. Some models do load correctly (like `dolphin-2.6-mistral-7b.Q2_K.gguf`) but won't even attempt to load some other models. Does this have a fix or they're simply not supported?

**Comment by Gee1111** [comment]
on my system every GGUF model is running with llama.cpp also the laser dolphin mixtrals
